cap/01-categorie.tex

```diff
@@ -1,9 +1,162 @@
 \chapter{Categorie, funtori, trasformazioni naturali}
+
+\section{Categorie}
+
+Prima di arrivare alla definizione formale di categorie, può essere di aiuto avere in mente un esempio concreto. Prendiamo per esempio gli spazi vettoriali e le mappe lineari tra di loro. 
+Molti di voi sapranno che le matrici invertibili su $\ictR^n$ con il loro prodotto formano un gruppo. Più un generale, se tralasciamo la condizione di invertibilità, abbiamo un \emph{monoide}:
+
+\begin{definition}
+ Un \emph{monoide} consiste in un insieme $M$ con:
+ \begin{itemize}
+  \item Un elemento particolare $e\in M$, chiamato \emph{elemento neutro}, \emph{unità} o \emph{identità};
```

Comment:
invece di "particolare" non è meglio "distinto"? Oppure qualcosa come "la scelta di un elemento $e\in M$, chiamato..."

Reply:
Corretto con "distinto" (vedi prossima pull request).